samples/compute/compute_vmss_example.ts

```diff
@@ -0,0 +1,713 @@
+            subnet = await this.createVirtualNetwork(this.resourceGroupName,this.location,this.networkName,this.subnetName);
+        }else{
+            subnet = "subneturi";
+        }
+
+        const parameter : compute.VirtualMachineScaleSet = {
+            sku: {
+                tier: "Standard",
+                capacity: 1,
+                name: "Standard_D1_V2"
+            },
+            location: "eastus",
+            overprovision: true,
+            virtualMachineProfile: {
+                storageProfile: {
+                    imageReference: {
+                        sku: "2016-Datacenter",
+                        publisher: "MicrosoftwindowsServer",
+                        version: "latest",
+                        offer: "windowsServer"
+                    },
+                    osDisk: {
+                        caching: "ReadWrite",
+                        managedDisk: {
+                            storageAccountType: "Standard_LRS"
+                        },
+                        createOption: "FromImage",
+                        diskSizeGB: 512
+                    }
+                },
+                osProfile: {
+                    computerNamePrefix: "testPC",
+                    adminUsername: "testuser",
+                    adminPassword: "000000000000000000",
+                },
+                networkProfile: {
+                    networkInterfaceConfigurations: [
+                        {
+                         name: "testPC",
+                         primary: true,
+                         enableIPForwarding: true,
+                         ipConfigurations: [
+                             {
+                                 name: "testPC",
+                                 subnet: {
+                                     id: "/subscriptions/" + subscriptionId + "/resourceGroups/" + this.resourceGroupName + "/providers/Microsoft.Network/virtualNetworks/" + this.networkName + "/subnets/" + this.subnetName + ""
+                                 } 
+                             }
+                         ]  
+                        }
+                    ]
+                }
+            },
+            upgradePolicy: {
+                mode: "Manual",
+                rollingUpgradePolicy: {
+                    maxUnhealthyUpgradedInstancePercent: 100,
+                    maxBatchInstancePercent: 20   // python is 100
+                }
+            }
+        }
+        // start an extension rolling upgrade.post
+        await this.compute_client.virtualMachineScaleSets.beginCreateOrUpdateAndWait(this.resourceGroupName,this.virtual_machine_scale_set_name,parameter).then(
+            response => {
+                console.log(response)
+            }
+        );
+    } 
+
+    //virtualMachineScaleSetVMs.getInstanceView
+    public async virtualMachineScaleSetVMs_getInstanceView(){
+        let instanceId: number;
```

Comment:
Is this the typical usage of virtualMachineScaleSetVMs_getInstanceView()?  @qiaozha

Reply:
no. there's no way to set the instanceId like this. 
@colawwj  Could you change it into call virtualMachineScaleSetVMs_List first the get all the available instanceIds and then use that instanceIds are returned from the result to call virtualMachineScaleSetVMs_getInstanceView

Reply:
changed